error_handling.ipynb

```diff
@@ -154,6 +181,59 @@
     "5. Write a function that takes a list of integers as an argument, and returns the largest integer in the list. Use a try-except block to catch any ValueError exceptions that may be raised when attempting to compare elements that are not integers."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 48,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "TypeError: 1.7 is a string. \n",
+      "Only integers list allowed.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# need to be completed\n",
+    "\n",
+    "def largest_int(lst): \n",
+    "        \n",
+    "    try:\n",
+    "        \n",
+    "        largest = None\n",
+    "           \n",
+    "        for i in lst:\n",
+    "            if largest is None or i > largest :\n",
+    "                largest = i\n",
+    "        \n",
+    "    except Exception as e:\n",
+    "        print('TypeError: {} is a string. '.format(i))\n",
+    "        print('Only integers list allowed.')\n",
+    "                \n",
+    "    else :\n",
```

Comment:
just remove `print statement` from `else block`. You are safe to delete `else statement` here

Reply:
done